src/style.css

```diff
@@ -0,0 +1,118 @@
+body {
+  background-image: url("../assets/pie2.jpeg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.container-fluid {
+  width: 100%;
+}
+
+body h3 {
+  font-family: var(--fredoka);
+  font-size: calc(2rem + 1vw);
+  font-weight: bold;
+  text-align: center;
+}
+
+.main {
+  background-color: rgba(32, 30, 30, 0.493);
+  color: white;
+}
+
+/* .menu-item {
+     display: grid;
+     -ms-grid-columns: auto auto;
+} */
+
```

Comment:
- [ ] It will be great to delete all the commented code in your project. This will keep your project clean and professional. Remember if you need to go to the previous version of your code you can always use git history